web/src/components/Review.test.js

```diff
@@ -1,7 +1,8 @@
-import { mount, shallow } from 'enzyme';
+import { shallow } from 'enzyme';
```

Comment:
do you still need shallow if you added rtl? also you can get fireEvent and screen from 'apd-testing-library'

Reply:
shallow is still used in other places within the test suite. Should I make another ticket about converting these test to rtl?